modules/configuring-aws-iam-for-cross-account-access.adoc

```diff
@@ -0,0 +1,115 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-10-19
+:_mod-docs-content-type: PROCEDURE
+
+[id="configuring-aws-iam-for-cross-account-access_{context}"]
+= Configuring AWS IAM for cross-account access
+
+You can configure {aws-short} Identity Access Management (IAM) for cross-account access.
+
+.Prerequisites
+* You have administrator permissions.
+* You have created the target role and the target role has permissions to send data to Cloudwatch or S3 outputs.
+* You have permissions to update the policy for the initial and the target role.
+
+.Procedure
+
+. Create an initial account in your cluster account to assume the target role with the required permissions.
++
+The initial account can either be a user or an IAM role.
+This account assumes the target role.
++
+[source,json]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": "sts:AssumeRole",
+            "Resource": "arn:aws:iam::<target-account>:role/target-logging-role"
+        }
+    ]
+}
+----
+
+. In your target role, create a trust policy and specify the initial cluster role, and optionally an external id.
++
+[source,json]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::<cluster-account>:<role_or_username>/initial-role"
```

Comment:
cluster-account to -- > aws-account-id  or account-id

Reply:
changed to aws-account-id